core/src/main/java/org/elasticsearch/action/admin/cluster/snapshots/status/SnapshotStatus.java

```diff
@@ -160,15 +159,7 @@ public static SnapshotStatus readSnapshotStatus(StreamInput in) throws IOExcepti
 
     @Override
     public String toString() {
-        try {
-            XContentBuilder builder = XContentFactory.jsonBuilder().prettyPrint();
-            builder.startObject();
-            toXContent(builder, EMPTY_PARAMS);
-            builder.endObject();
-            return builder.string();
-        } catch (IOException e) {
-            return "{ \"error\" : \"" + e.getMessage() + "\"}";
-        }
+        return Strings.toString(this, true, true);
```

Comment:
Since the original toString() method only used the `pretty` printing option but not the 'human` option, could we keep that the same?

Reply:
Make sense. Thanks.